src/components/ClaimMasterPanel.js

```diff
@@ -81,6 +81,7 @@ class ClaimMasterPanel extends FormPanel {
     this.isClaimedDateFixed = props.modulesManager.getConf("fe-claim", "claimForm.isClaimedDateFixed", false);
     this.EMPTY_STRING = "";
     this.showPreAuthorization = props.modulesManager.getConf("fe-claim", "showPreAuthorization", false);
+    this.guaranteeNo = props.modulesManager.getConf("fe-claim", "guaranteeNo", "O");
```

Comment:
Instead of using shorthand values like "M", "O", and "N", define an object with more descriptive constant names and use it directly in the code:

`const GUARANTEE_NO = {
MANDATORY: "M",
OPTIONAL: "O",
HIDDEN: "N",
};`

This makes the code self-explanatory, so no need for additional explanations in the README.

Reply:
I can't create constants like this because we will have many fields like this